### PR TITLE
Add links from deprecated labels to their replacements

### PR DESCRIPTION
**Pull request by wafuwafu13**

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
lgtm

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -405,13 +405,13 @@ For more details, see [Addon-manager](https://github.com/kubernetes/kubernetes/b
...
 
-This label has been deprecated. Please use `kubernetes.io/arch` instead.
+This label has been deprecated. Please use [kubernetes.io/arch](#kubernetes-io-arch) instead.
 
 ### beta.kubernetes.io/os (deprecated)
 
 Type: Label
 
-This label has been deprecated. Please use `kubernetes.io/os` instead.
+This label has been deprecated. Please use [kubernetes.io/os](#kubernetes-io-os) instead.
```
Comment (jihoon-seo) [inline content/en/docs/reference/labels-annotations-taints/_index.md:414]: How about preserving backticks?

```diff
-This label has been deprecated. Please use [kubernetes.io/arch](#kubernetes-io-arch) instead.
+This label has been deprecated. Please use [`kubernetes.io/arch`](#kubernetes-io-arch) instead.

-This label has been deprecated. Please use [kubernetes.io/os](#kubernetes-io-os) instead.
+This label has been deprecated. Please use [`kubernetes.io/os`](#kubernetes-io-os) instead.
```

Ref:

- https://github.com/kubernetes/website/blob/main/content/en/docs/reference/labels-annotations-taints/_index.md?plain=1#L1067
```markdown
[`kubectl.kubernetes.io/default-container`](#kubectl-kubernetes-io-default-container)
```

- https://github.com/kubernetes/website/blob/main/content/en/docs/reference/labels-annotations-taints/_index.md?plain=1#L1637
```markdown
see [`service.beta.kubernetes.io/aws-load-balancer-scheme`](#service-beta-kubernetes-io-aws-load-balancer-scheme).
```

...

**Review by sftim — COMMENTED** [review]
/lgtm

Backticks are even better.

**Review by sftim — COMMENTED** [review]
Thanks

/retitle Add links from deprecated labels to their replacements
/lgtm
/approve

**Pull request merged**